rs/ledger_suite/tests/sm-tests/src/fee_collector.rs

```diff
@@ -199,15 +199,20 @@ pub fn test_fee_collector_blocks<T>(
                 arb_account(),
                 arb_account(),
                 arb_account(),
+                arb_account(),
                 1..10_000_000u64,
             )
-                .prop_filter("The three accounts must be different", |(a1, a2, a3, _)| {
-                    a1 != a2 && a2 != a3 && a1 != a3
-                }),
-            |(account_from, account_to, fee_collector_account, amount)| {
+                .prop_filter(
+                    "The three accounts must be different",
+                    |(a1, a2, a3, a4, _)| {
+                        a1 != a2 && a2 != a3 && a1 != a3 && a1 != a4 && a2 != a4 && a3 != a4
```

Comment:
This is a bit hard to manage (as in "visually checking it's correct and later make changes to it").
Could we do something like: `|(a1, a2, a3, a4, _)| HashSet::from([a1, a2, a3, a4]).len() == 4` ?

Reply:
Great suggestion, thanks - fixed!